compiler/src/iree/compiler/Codegen/Passes.h

```diff
@@ -552,6 +552,14 @@ std::unique_ptr<OperationPass<ModuleOp>> createConvertToNVVMPass();
 /// Performs the final conversion to ROCDL+LLVM dialect.
 std::unique_ptr<OperationPass<ModuleOp>> createConvertToROCDLPass();
 
+/// A Pass to lowers linalg's fill and matmul to microkernel op
+std::unique_ptr<OperationPass<func::FuncOp>> createLLVMGPULowerToUKernelsPass();
+
+/// Lowers linalg's fill and matmul to microkernel op
+FailureOr<Operation *> lowerMatmulToMicrokernel(
```

Comment:
does this need to be exposed this way? This is GPU specific transformation.... There is a similar matcher for CPU, so it can be confusing.

I need to split this `Passes.h` into 4 parts.....

Reply:
I have implemented transform dialect op for microkernel that calls `lowerMatmulToMicrokernel` for transformation (that will come as next PR :)) .

I agree that it is GPU specific, how about I change its name to `lowerGPUMatmulToMicrokernel`?

Reply:
```I need to split this Passes.h into 4 parts.....```
This makes sense to me. I am happy to do it if you are busy :)

Reply:
If you are willing to do it, I'll be very happy!